### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by iamalhera**

### CODE REVIEW

File: src/pages/common.css [hunk 1]
```diff
@@ -196,20 +208,83 @@
...
+.list{
+    display : flex;
+    flex-direction : column;
+    justify-content: space-between ;
+    flex-wrap : wrap;
+    padding: 1.5rem ;
+    border-radius: 5px;
+}
+.list-item{
+    font-size : 0.95rem ;
```
Comment (yaswanthmyneni) [inline src/pages/common.css:244]: changes: use px units for less than 1rem.

File: src/pages/common.css [hunk 2]
```diff
@@ -196,20 +208,83 @@
...
 
-/* button */
-.btn-primary{
-    background-color: var(--ca-primary-color);
-    color : white ;
+/* card */
+.card {
+    font-family: 'Montserrat', sans-serif;
+    border-radius: 10px;
+    box-shadow: 0px 1px 10px rgb(146 140 140 / 50%);
```
Comment (yaswanthmyneni) [inline src/pages/common.css:254]: changes: use CSS variables for colors.

File: src/pages/common.css [hunk 3]
```diff
@@ -196,20 +208,83 @@
...
+    margin : 1.5rem ;
 }
-.btn-primary:hover{
-    background-color: var(--ca-primary-color-lighter);
-    color : white ;
-}
\ No newline at end of file
+.card-vertical {
+    width: 300px;
+    height: 450px;
```
Comment (yaswanthmyneni) [inline src/pages/common.css:259]: suggestion: you can use the rem unit here. If you don't want it to be relative than it is fine.

File: src/pages/product-page/product-page.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
+@import url('/src/pages/homepage.css');
+@import url('/src/pages/common.css');
+.container-header{
+    grid-area: header;
+}
+ .product-page-main-container{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
```
Comment (yaswanthmyneni) [inline src/pages/product-page/product-page.css:8]: suggestion: you can use repeat() here like repeat(5, 1fr).

File: src/pages/product-page/product-page.html [hunk 5]
```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
```
Comment (yaswanthmyneni) [inline src/pages/product-page/product-page.html:1]: UI improvement: you can place the shop icon on the right side of the header just beside the profile icon.
Screenshot 2022-02-21 at 9 54 11 AM

**Review by yaswanthmyneni — APPROVED** [review]
UI looks good, you can merge after changes.

File: styles.css [hunk 6]
```diff
@@ -4,6 +4,7 @@
     display: flex;
     align-items: center;
     justify-content: space-between;
+    height : 80px;
```
Comment (vanshitamoolchandani) [inline styles.css:7]: Change it to rem.

**Review by vanshitamoolchandani — APPROVED** [review]

File: src/pages/common.css (hunk 3 above)
Comment (iamalhera) [reply src/pages/common.css:259]: i want this as a standard size

File: src/pages/product-page/product-page.html (hunk 5 above)
Comment (iamalhera) [reply src/pages/product-page/product-page.html:1]: will look into it

File: src/pages/product-page/product-page.css (hunk 4 above)
Comment (iamalhera) [reply src/pages/product-page/product-page.css:8]: right

File: styles.css (hunk 6 above)
Comment (iamalhera) [reply styles.css:7]: sure, thanks for suggestion

**Pull request merged**